# name: logging.exception 
logging.exception(msg, *args, **kwargs)